src/components/Header.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from "prop-types";
+
+const Header = ({ text, highlightWords, size, textColor, textAlign }) => {
+  const renderText = () => {
+    const words = text.split(" ");
+    return words.map((word, index) => {
+      const isHighlighted = highlightWords.includes(word);
+
+      const highlightClass = isHighlighted
+        ? "underline decoration-wavy decoration-orange-500 font-bold"
+        : "font-bold";
+
+      return (
+        <span key={index} className={highlightClass}>
+          {word}{" "}
+        </span>
+      );
+    });
+  };
+
+  const headerClass = `${size === "h1" ? "text-6xl" : size === "h2" ? "text-3xl" : "text-2xl"} text-${textColor} ${textAlign} font-bold`;
```

Comment:
Statement like these should be broken into multiple meaningful likes or maybe using switch-case/if-else because at some point they do become confusing.

Reply:
Statements have been broken down to switch case now. Thanks for the suggestion @rt4914 
Header UI `( case h1, h2, h3 )`:
![image](https://github.com/user-attachments/assets/e5fb18ca-2be3-45a4-838f-57f8039b5856)

Storybook header variants:
![image](https://github.com/user-attachments/assets/b0aa0e98-777a-4df5-96e9-e9075e78cd88)
![image](https://github.com/user-attachments/assets/6f70e7b9-69bc-4960-beb1-ee8ee80d90a8)
![image](https://github.com/user-attachments/assets/6c098ef8-301e-4880-8495-4afca88cec55)

Reply:
I have updated the description. 
You should always add these screenshots to description because this comment will get resolved and after that the UI won't be visible.